src/components/Footer.tsx

```diff
@@ -0,0 +1,207 @@
+import { css, useTheme } from '@emotion/react';
+import styled from '@emotion/styled';
+import Link from 'next/link';
+import React, { FC } from 'react';
+
+import { LogoIcon } from './icons/LogoIcon';
+import { onlyXs, atMinMd, atMinLg } from '~/theme';
+import { Container } from './Container';
+import { Heading } from '~/components';
```

Comment:
This is actually causing a build error because of a circular dependency. Rule of thumb from Drew is to always import peers directly instead of going "up" to the index.
```suggestion
import { Heading } from './Heading';
```